pkg/porter/explain_test.go

```diff
@@ -259,6 +280,33 @@ func TestExplain_generatePrintableBundleCreds(t *testing.T) {
 	assert.Equal(t, 0, len(pb.Actions))
 }
 
+func TestExplain_generatePrintableBundlePorterVersion(t *testing.T) {
+	bun := bundle.Bundle{
+		Definitions: definition.Definitions{
+			"string": &definition.Schema{
+				Type:    "string",
+				Default: "clippy",
+			},
+		},
+		Custom: map[string]interface{}{
+			"sh.porter": map[string]interface{}{
+				"manifest": "5040d45d0c44e7632563966c33f5e8980e83cfa7c0485f725b623b7604f072f0",
+				"version":  "v0.30.0",
+				"commit":   "3b7c85ba",
+			},
+		},
+	}
+
+	pb, err := generatePrintable(bun)
+	assert.NoError(t, err)
+
+	assert.Equal(t, "v0.30.0", pb.PorterVersion)
+	assert.Equal(t, 0, len(pb.Parameters))
```

Comment:
For porter tests, it's okay to not assert for things unrelated to the test, so it's okay to just test `pb.PorterVersion` and not all the other fields in this test.